hail/python/hailtop/batch_client/aioclient.py

```diff
@@ -342,6 +345,18 @@ def create_job(self, image, command=None, args=None, env=None, ports=None,
         self._jobs.append(j)
         return j
 
+    async def _submit_job_with_retry(self, batch_id, docs):
+        n_attempts = 0
+        saved_err = None
+        while n_attempts < max_job_submit_attempts:
+            try:
+                return await self._client._post(f'/api/v1alpha/batches/{batch_id}/jobs/create', json={'jobs': docs})
+            except Exception as err:  # pylint: disable=W0703
+                saved_err = err
+                n_attempts += 1
+                await asyncio.sleep(0.5)
```

Comment:
check which ones have already been made then send the diff

Reply:
This isn't necessary. None are committed to the database if any in the batch fail.